client/app/src/pages/admin/backup/backup-tab1/backup-tab1.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container>
+    <div class="col-md-6">
+        <div class="form-group">
+            <label for="backup-enable">{{ 'Enable Backup' | translate }}</label>
+            <input type="checkbox" [(ngModel)]="backupEnable" id="backup-enable" />
+        </div>
+    
+        <div class="form-group">
+            <label for="backup-time">{{ 'Backup Time (hh:mm)' | translate }}</label>
+            <input type="time" [(ngModel)]="backupTime" id="backup-time" class="form-control" />
+        </div>
+    
+        <div class="form-group">
+            <label for="backup-destination">{{ 'Backup Destination Path' | translate }}</label>
+            <input type="text" [(ngModel)]="backupDestinationPath" id="backup-destination" class="form-control" />
+        </div>
+
+        <div class="form-group">
+            <button class="btn btn-primary" id="save_settings" [disabled]="!backupEnable" (click)="save()" type="submit">
+                <i class="fa-solid fa-check"></i>
+                <span>{{ 'Save' | translate }}</span>
+            </button>
+        </div>
+    </div>
+</ng-container>
```

Comment:
It seems currently not possible to prepare the configuration of the Backups without enabling the functionality.

In the context of GlobaLeaks we would probably change this aspect.

![image](https://github.com/user-attachments/assets/16e83a58-6c19-4168-b5d7-5b09ad2ffbf8)

In addition right know it seems that the Backup section is shown also to administrators of secondary tenants.
The feature should be probably be available only on the primary tenant.